automated/utils/test-runner.py

```diff
@@ -16,9 +16,15 @@
 import time
 from uuid import uuid4
 from distutils.spawn import find_executable
-from squad_client.core.api import SquadApi
-from squad_client.shortcuts import submit_results
-from squad_client.core.models import Squad
+
+
+try:
+    from squad_client.core.api import SquadApi
+    from squad_client.shortcuts import submit_results
+    from squad_client.core.models import Squad
+    from urllib.parse import urlparse
+except ImportError as e:
+    print('squad_client is needed if you want to upload to qa-reports')
```

Comment:
please use logging for printing messages.

Reply:
Fixed it.